etc/firebase-admin.auth.api.md

```diff
@@ -264,9 +264,19 @@ export interface ListUsersResult {
     users: UserRecord[];
 }
 
+// @public
+export class MultiFactorAuthConfig implements MultiFactorConfig {

```

Comment:
Good point. Yes these types don't exist in the API proposal. Should we update it? @prameshj @Xiaoshouzi-gh @renkelvin

Reply:
Yes, we should stick to the API proposals.

Maybe just make MultiFactorAuthConfig internal?

Reply:
to make it internal, please check `SmsRegionsAuthConfig` for reference.

Reply:
This class isn't user facing. It just implements the MultiFactorConfig interface which is public. @lahirumaramba should i remove /** */ documentation to undo this?

Reply:
Since we export it, it will still show up in the public API. I think we can mark it as `@internal` similar to here: https://github.com/firebase/firebase-admin-node/blob/81672d6e67bf36008fd9ecd4921e7ed4269b0ff6/src/auth/auth-config.ts#L1520

Reply:
Updated.